lib/features/support/env.rb

```diff
@@ -21,6 +21,7 @@
 Before do
   stored_requests.clear
   find_default_docker_compose
+  clear_docker_services
```

Comment:
My understanding is that this will execute code which is specific to Docker on every mazerunner run, when lots of the platforms don't use Docker. Are there good reasons why we need to do it this way?

Reply:
Good question - The answer is that we shouldn't.  Those functions should be in the env file of the project that uses them.  I'll remove both `clear_docker_services` and `find_default_docker_compose`.  @martin308 this may be a breaking change for you?

Reply:
I think it should be ok. It looks like I'm going to need to write a whole bunch of custom stuff anyway due to the way docker works/doesn't work on windows